benchmark/fft_batch/fft_batch_config.cc

```diff
@@ -0,0 +1,55 @@
+#include "benchmark/fft_batch/fft_batch_config.h"
+
+#include "tachyon/base/console/iostream.h"
+#include "tachyon/base/containers/container_util.h"
+#include "tachyon/base/flag/flag_parser.h"
+
+namespace tachyon::benchmark {
+
+bool FFTBatchConfig::Parse(int argc, char** argv) {
+  base::FlagParser parser;
+
+  // clang-format off
+  parser.AddFlag<base::Flag<std::vector<size_t>>>(&exponents_)
+      .set_short_name("-k")
+      .set_required()
+      .set_help("Specify the exponent 'k's where the degree of poly to test is 2ᵏ.");
+  parser.AddFlag<base::BoolFlag>(&run_coset_lde_)
+      .set_long_name("--run_coset_lde")
+      .set_default_value(false)
+      .set_help("Run CosetLDE benchmark. Default is FFT benchmark.");
+  parser.AddFlag<base::Flag<bool>>(&check_results_)
+      .set_long_name("--check_results")
+      .set_help("Whether checks results generated by each fft_batch runner.");
+  parser.AddFlag<base::Flag<size_t>>(&batch_size_)
+      .set_short_name("-b")
+      .set_long_name("--batch_size")
+      .set_default_value(100)
+      .set_help("Specify the batch size. By default, 100.");
+  parser.AddFlag<base::Flag<FieldType>>(&prime_field_)
+      .set_short_name("-p")
+      .set_long_name("--prime_field")
+      .set_default_value(FieldType::BabyBear())
+      .set_help(
+          "A prime field to be benchmarked with. (supported prime fields: "
+          "baby_bear");
+  parser.AddFlag<base::Flag<std::vector<Vendor>>>(&vendors_)
+      .set_long_name("--vendor")
```

Comment:
Do you allow it to run fft batch only with tachyon? Otherwise, please add `set_required()`

```suggestion
      .set_long_name("--vendor")
      .set_required()
```

Reply:
I think we can run only tachyon sometimes.